packages/lesswrong/components/common/ContentItemTruncated.tsx

```diff
@@ -39,15 +43,33 @@ const ContentItemTruncated = ({classes, maxLengthWords, graceWords=20, expanded=
       wordsLeft: 0,
     } : truncateWithGrace(html, maxLengthWords, graceWords, rawWordCount);
   
+  useEffect(() => {
```

Comment:
FYI: this is fixing some pre-existing broken behavior in tag revision histories, where too-long content would get truncated without a "show more".